about.html

```diff
@@ -0,0 +1,109 @@
+        <a class="thecd" href="https://www.amazon.com/Way-Red-Hot-Chili-Peppers/dp/B000067CPX/ref=sr_1_2?crid=NJUK25R28D4O&keywords=by+the+way+cd&qid=1649808283&sprefix=by+the+wa+cd%2Caps%2C162&sr=8-2"><img src="./assets/Images/album7.jpg" alt="by he way"></a>
+        <a class="thecd" href="https://www.amazon.com/Californication-Red-Chili-Peppers-Audio/dp/B00FT15NXE/ref=sr_1_1?crid=R5EQR2D2RTGB&keywords=californication+cd&qid=1649808262&sprefix=californication+cd%2Caps%2C159&sr=8-1"><img src="./assets/Images/album6.jpg" alt="californication"></a>
+        <a class="thecd" href="https://www.amazon.com/One-Hot-Minute-Chili-Peppers/dp/B000002MTR/ref=sr_1_1?crid=2DXXICCNCFZ61&keywords=one+hot+minute+cd&qid=1649808240&sprefix=one+hot+minute+c%2Caps%2C180&sr=8-1"><img src="./assets/Images/album5.jpg" alt="one hot minute"></a>
+        <a class="thecd" href="https://www.amazon.com/Blood-Sugar-Magik-Chili-Peppers/dp/B000002LQR/ref=sr_1_1?keywords=blood+sugar+sex+magik+cd&qid=1649808199&sprefix=blood+sugar+se%2Caps%2C210&sr=8-1"><img src="./assets/Images/album4.jpg" alt="blood sugar sex magik"></a>
+        <a class="thecd" href="https://www.amazon.com/Mothers-Milk-Explicit-Chili-Peppers/dp/B000078DOI/ref=sr_1_1?crid=S2G0VZ0O8VEZ&keywords=mothers+milk+cd&qid=1649858489&sprefix=mothers+milk+cd%2Caps%2C186&sr=8-1"><img src="./assets/Images/albumX.jpg" alt="mothers milk"></a>
+        <a class="thecd" href="https://www.amazon.com/Uplift-Mofo-Party-Plan-Explicit/dp/B000078DOJ/ref=sr_1_1?keywords=uplift+mofo+party+plan+cd&qid=1649808151&sprefix=uplift+mofo%2Caps%2C178&sr=8-1"><img src="./assets/Images/album3.jpg" alt="the uplift mofo party plan"></a>
+        <a class="thecd" href="https://www.amazon.com/Freaky-Styley-Explicit-Chili-Peppers/dp/B000078DOH/ref=sr_1_1?keywords=freaky+styley+cd&qid=1649808126&sprefix=freaky+sty%2Caps%2C170&sr=8-1"><img src="./assets/Images/album2.jpg" alt="freaky styley"></a>
+        <a class="thecd" href="https://www.amazon.com/Red-Hot-Chili-Peppers-Explicit/dp/B000078DOK/ref=sr_1_10?crid=20AICI1OMUODH&keywords=the+red+hot+chili+peppers+album&qid=1649808086&sprefix=the+red+hot+chili+peppers+albu%2Caps%2C189&sr=8-10"><img src="./assets/Images/album1.jpg" alt="self-titled"></a>
+      
+
+       </div>
+       </section>
+
+           <!-- Partners -->
+
+      <section class="partners" id="sponsors">
+        <h2>Partners</h2>
+       <hr class="divisionline1"/>
+        <div class="brandlogos">
+          
+          <img src="./assets/Images/sponsor1.png" alt="macdonalds">
+          <img src="./assets/Images/sponsor2.png" alt="adidas">
+          <img src="./assets/Images/sponsor3.png" alt="cocacola">
+          <img src="./assets/Images/sponsor4.png" alt="apple">
+
+          
+        </div>
+      </section>
```

Comment:
- [ ] Kindly the `Partners` section doesn't appear on the about page on the desktop version, appears on the mobile version.

Your design
![image](https://user-images.githubusercontent.com/35535130/163290229-69d5dca1-3670-41d7-9009-bab9f81f368e.png)

Original design
![image](https://user-images.githubusercontent.com/35535130/163290319-4f05362f-a6ce-4296-8675-ce042f2c0f89.png)

Reply:
Just fixed this part as well, thanks!